Week-2/Assignment-2/index.js

```diff
@@ -0,0 +1,26 @@
+console.log('Assignment 2');
+
+
+const containerElement = document.getElementById("box__container");
+
+const redBox = document.getElementById("red")
+const yellowBox = document.getElementById("yellow")
+const blueBox = document.getElementById("blue")
+const greenBox = document.getElementById("green")
+
+
+containerElement.addEventListener("mouseover",function(event){
```

Comment:
Sakar, you have achieve the  transformation withoyt using JS, only HTML and CSS is allowed.

Reply:
Implemented in css .